pkg/my/domain.go

```diff
@@ -59,13 +60,61 @@ func Websites(client *node.Client, domainNames []string) ([]*models.Websites, er
 		return nil, fmt.Errorf("reading entries'%s': %w", params, err)
 	}
 
-	for i := 0; i < len(domainNames); i++ {
+	for i := 0; i < len(domainNames); i++ { //nolint:varnamelen
+		contractAddress := string(contractAddresses[i].CandidateValue)
+
+		hasBrokenChuck, err := checkChunkIntegrity(client, contractAddress)
+		if err != nil {
+			return nil, fmt.Errorf("checking chunk integrity: %w", err)
+		}
+
 		response := models.Websites{
-			Address: string(contractAddresses[i].CandidateValue),
-			Name:    domainNames[i],
+			Address:        contractAddress,
+			Name:           domainNames[i],
+			HasBrokenChunk: hasBrokenChuck,
 		}
 		responses = append(responses, &response)
 	}
 
 	return responses, nil
 }
+
+// Check website chunks and store its ID in an array if one of them is broken.
+func checkChunkIntegrity(client *node.Client, address string) ([]string, error) {
```

Comment:
should we call this function something like `getMissingChunkIds`?

should this function return an array of integer?

Reply:
ok for me.

For later improving this, I would suggest getMissingChunkIds to return array of integers and in models.Websites transform into array of string. Because getMissingChunkIds could be use for something else.